### ISSUE TITLE
VNAV never showing ~~PATH~~ (strike-through) when unable to find path but always staying armed

### ISSUE THREAD

**Issue by okurz** [Working-Title-MSFS-Mods/fspackages#906]
## Observation

I think I have never seen ~~PATH~~ displayed for VNAV and it seems if I am above any possible VNAV descent path the VNAV debug monitor page shows no alt dev or TOD computation. Only If I descend fast and steep enough eventually it will switch to "INTERCEPTED" and the annunciation VPATH is activated. Can anyone confirm they can see ~~PATH~~ ?

I hope the video https://www.youtube.com/watch?v=RcbAA6x7GDg can visualize the issue.

## Expected result

```
[1:40 PM] Windhover [Z-6]: You'll see PATH with a strikethrough whenever you are too high for VNAV to capture a vertical path.
[1:42 PM] diesel: As I was stating, I never saw PATH in the cockpit. I only see it described in documentation. So I wonder how I can find out if I am actually too high for a path to be captured
[2:39 PM] Matt (nishmaster) [Z-5]: You should get ~PATH~ if you are more than 1000ft high, I believe
```

## Problem

I assume someone has added that ~~PATH~~ and knows under which cirumstances this actually works or if it still works. I would be interested in crosschecking the "happy path" that was used for testing agains the use case I had.

**Comment by K20017** [comment]
Investigating. Does this still occur on 0.11.2?

**Comment by okurz** [comment]
Please see

![fs2020_cj4_0 11 2_above_VNAV_PATH](https://user-images.githubusercontent.com/1693432/111277278-91f56080-8638-11eb-8c0a-8566779ad86e.jpg)

The plane is holding FL434 as seen on PFD. MFD shows that TOD has passed, difference -14k FPM, but PFD shows that VNAV is armed, "PATH" is armed but no "NOPATH" nor ~~PATH~~ . So can you tell, what should be expected? Maybe I am using the system wrong or having wrong expectations. Other than that VNAV works fine :)

**Comment by okurz** [comment]
I can confirm that 0.11.3 behaves quite different and I can easily trigger a ~~PATH~~ indication where it should be expected. So you can consider this issue fixed in 0.11.3 andcall the issue closed depending on if you want to have a stable release declared or enough to have it fixed in the beta version.
